vignettes/.translation_links.R

```diff
@@ -0,0 +1,32 @@
+# build a link list of alternative languages
+# currently inline IETF codes but can be expanded
+url = "https://rdatatable.gitlab.io/data.table/articles"
+path = dirname(knitr::current_input(TRUE))
+
+if (basename(path) == "vignettes") {
+  lang = "en"
+} else {
+  lang = basename(path)
+  path = dirname(path)
+}
+
+translation = dir(path,
+  recursive = TRUE,
+  pattern = glob2rx(knitr::current_input(FALSE))
+)
+
+transl_lang = ifelse(dirname(translation) == ".", "en", dirname(translation))
+
+block = sprintf(
+  "%s: %s\n", 
+  switch(lang,
+    fr = "Des traductions de ce document sont disponibles dans les langues suivantes",
```

Comment:
This method for defining translations is non-standard. I think this would be better as gettext(...)
or to simplify, just use English. Only list the other languages at the top of the English vignette. and then the other vignettes can just link back to English.

Reply:
Yes, I think in gettext however I don't figure out how to get the string for the appropriate language to compile the vignettes in all languages, other than set `Sys.setLanguage()` in each call.
Another option is to hardcode the header and let the code generate only the links.
But I think your second option is ok.